generators/app/templates/gulp/templatesFunctions.js

```diff
@@ -42,7 +43,16 @@ module.exports = function(data) {
         }
         return classes.join(' ');
       }
-    }
+    },
+    {
+      name: 'hasVendor',
+      func: function () {
+        var fileListPath = pathModule.join(data.config.src.base, 'scripts/vendor.json');
```

Comment:
Can we use exact same logic as in php version for consistency? Now for example when you put file that doesn't exist in json this returns true but vendor is not created.